docs/tables/yml_key_value.md

```diff
@@ -0,0 +1,242 @@
+| receipt              | Oz-Ware Purchase Invoice                                                                          | string     | []                          | 2          |
+| items.0.quantity     | 4                                                                                                 | integer    | []                          | 12         |
+| items.0.description  | Water Bucket (Filled)                                                                             | string     | []                          | 10         |
+| city                 | East Centerville                                                                                  | string     | []                          | 22         |
+| items.1.quantity     | 1                                                                                                 | integer    | []                          | 17         |
+| state                | KS                                                                                                | string     | []                          | 23         |
+| bill_to              | <null>                                                                                            | null       | []                          | 18         |
+| street               | 123 Tornado Alley                                                                                 | string     | []                          | 19         |
+|                      | Suite 16                                                                                          |            |                             |            |
++----------------------+---------------------------------------------------------------------------------------------------+------------+-----------------------------+------------+
+```
+
+or, you can query configurations of a particular file using:
+
+```sql
+select
+  key_path,
+  value,
+  tag as value_type,
+  pre_comments,
+  start_line
+from
+  yml_key_value
+where
+  path = '/Users/myuser/yml/invoice.yml';
+```
+
+## Examples
+
+### Query a simple file
+
+Given the file `invoice.yml` with following configuration:
+
+```yml
+---
+receipt: Oz-Ware Purchase Invoice
+date: 2012-08-06
+customer:
+    first_name:  Dorothy
+    family_name: Gale
+# List of ordered items
+items:
+  - part_no: A4786 # item 1
+    description: Water Bucket (Filled)
+    price: 1.47
+    quantity: 4
+  - part_no: E1628 # item 2
+    description: High Heeled "Ruby" Slippers
+    size: 8
+    price: 133.7
+    quantity: 1
+bill-to: &id001
+street: |
+  123 Tornado Alley
+  Suite 16
+city: East Centerville
+state: KS
+ship-to: *id001
+specialDelivery: >
+  Follow the Yellow Brick
+  Road to the Emerald City.
+  Pay no attention to the
+  man behind the curtain.
+```
+
+and the query is:
```

Comment:
Since we have a section dedicated to ltree searching, my suggestion would be to not use any ltree functions until that section, instead keep these examples simpler